metadata_service/proxy/gremlin_proxy.py

```diff
@@ -170,8 +173,14 @@ def get_table(self, *, table_uri: str) -> Table:
                by(self.key_property_name).\
                by('partition_key').\
                by('partition_value').\
-               by('create_time').fold()
-            ).next()
+               by('create_time').fold()).\
+            by(__.out('DERIVED_FROM').\
+               project('application_id', 'name', 'description', 'application_url')
+               .by(self.key_property_name)
+               .by("name")
+               .by("description")
+               .by(__.coalesce(__.values("application_url"), __.constant(''))).fold()).\
```

Comment:
Is there a better way to allow _any_ of these fields to be empty? It seemed to interpret a `''` as "missing" and threw an exception (as tested in one of the unit tests)... but I didn't completely understand why. Ideally all of these would default to be `None`.